spec/system/publishers_can_manage_job_applications_for_a_vacancy.rb

```diff
@@ -0,0 +1,218 @@
+require "rails_helper"
+
+RSpec.describe "Publishers can manage job applications for a vacancy" do
+  let(:organisation) { create(:school, name: "A school with a vacancy") }
+  let(:publisher) { create(:publisher) }
+
+  before { login_publisher(publisher: publisher, organisation: organisation) }
+
+  context "when a vacancy has expired and it has applications" do
+    let(:vacancy) { create(:vacancy, :expired) }
+    let!(:job_application_submitted) { create(:job_application, :status_submitted, vacancy: vacancy) }
+    let!(:job_application_shortlisted) { create(:job_application, :status_shortlisted, vacancy: vacancy) }
+    let!(:job_application_unsuccessful) { create(:job_application, :status_unsuccessful, vacancy: vacancy) }
+    let!(:job_application_draft) { create(:job_application, :status_draft, vacancy: vacancy) }
+
+    before do
+      vacancy.organisation_vacancies.create(organisation: organisation)
```

Comment:
This could be part of line 10:
```
let(:vacancy) { create(:vacancy, :expired, organisation_vacancies_attributes: [{ organisation: organisation }]) }
```